spec/controllers/application_controller/feature_spec.rb

```diff
@@ -1,25 +1,21 @@
-require File.expand_path("../app/controllers/application_controller/", "feature.rb")
-
 describe ApplicationController do
```

Comment:
Yeah, I think i'm just going to leave this one alone for now.  I tried a few alternatives, but this doesn't look as bad as it was.